feature/experimental/isis/ate_tests/base_adjacencies_test/base_adjacencies_test.go

```diff
@@ -262,7 +278,7 @@ func TestBasic(t *testing.T) {
 				}
 			})
 		}
-		deadline = time.Now().Add(time.Second)
+		deadline = time.Now().Add(time.Minute)
```

Comment:
same, why 1 min.

Reply:
each rpc needs 3 secs, if multiple rpc calls are in this block of code then we need to increase that duration